hivemind/averaging/matchmaking.py

```diff
@@ -477,37 +475,31 @@ def request_expiration_time(self) -> float:
         else:
             return min(get_dht_time() + self.averaging_expiration, self.search_end_time)
 
-    async def _update_queue_periodically(self, key_manager: GroupKeyManager):
```

Comment:
The code is identical besides removing the `try-except` block (rationale is explained in [this](https://github.com/learning-at-home/hivemind/pull/357/files#r695271601) comment).